src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -386,7 +386,7 @@
       [modal/Component {:ref "modal"}]])
    :component-did-mount
    (fn [{:keys [this state refs]}]
-     (set! utils/auth-expiration-handler #(sign-in/show-sign-in-dialog :expired))
+     (set! utils/auth-expiration-handler (partial sign-in/show-sign-in-dialog :expired))
```

Comment:
This was a pain-in-the-ass to debug. I'd like us to do a quick huddle on how we could improve the situation. There may be no answer, but I'd at least like to have the conversation.